### DISCUSSION TITLE
How to check if SharePoint site already Teamified before running New-PnPTeamsTeam command?

### DISCUSSION THREAD

**Discussion by khha** [pnp/powershell#3597] [Q&A]
I would like to run the command "New-PnPTeamsTeam", but it looks like this command is also working when a SharePoint site already is Teamified. It takes allot of time. I would like to skip this command if the SharePoint site already is teamified. How can I check if a SharePoint site is already Teamified?

**Comment by reshmee011** [comment]
Get-PnPTeamsTeam might help to identify whether the SharePoint site is already teamified.
